examples/tensorflow/text-classification/run_text_classification.py

```diff
@@ -399,6 +361,11 @@ def preprocess_function(examples):
         return result
 
     datasets = datasets.map(preprocess_function, batched=True, load_from_cache_file=not data_args.overwrite_cache)
+
+    if data_args.pad_to_max_length:
+        data_collator = partial(default_data_collator, return_tensors="tf")
+    else:
+        data_collator = DataCollatorWithPadding(tokenizer, return_tensors="tf")
```

Comment:
an alternative to `return_tensors="tf"` is to set the collator args in `to_tf_dataset()`, but I think this way is more user-friendly